molo/core/tests/test_utils.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from molo.core import utils
+
+
+class TestUtils(TestCase):
+
+    def test_get_locale_code(self):
+        assert utils.get_locale_code() == 'en'
```

Comment:
Slight preference for `assertEquals()` as it's slightly better practice in tests and generates a more useful error message when it fails.